apps/emqx_bridge/src/emqx_bridge_schema.erl

```diff
@@ -80,6 +80,9 @@ direction_field(Dir, Desc) ->
 
 %%======================================================================================
 %% For config files
+
+namespace() -> "bridge".
```

Comment:
Q: what happens when there are more than one module with the same namespace?  They get merged together?

Reply:
Good question.
Currently, last one wins.
https://github.com/emqx/hocon/blob/8e25f312507b103b06a794d85dcdf2f65e59dea4/src/hocon_schema.erl#L255
Would be better if we add a check to see if the fields are actually the same, and crash otherwise